### ISSUE TITLE
This repository is empty. Care to check out the GitHub Channel on YouTube while you wait?

### ISSUE THREAD

**Issue by 10Dev** [vatlab/sos-dataexchanger#5]
There should be a readme.md to explain where the code for this project is located, specially now that @BoPeng has mentioned it in https://github.com/jupyterlab/jupyterlab/issues/5548

**Comment by BoPeng** [comment]
This repository is in the planning stage.  We did a lot of brainstorming in the format of private emails and tried to determine the scope of the project. The proposal I described in the databus ticket is one of the most mature ones. This repository was created to host some experimental code but we never got enough time to do it.

**Comment by 10Dev** [comment]
I'm really just suggesting that you take what you have and dump it in the Wiki or Readme or whatever. That is kind of the OSS way of things...

It is just a bit odd when you mention a project and then when I look it up, there is absolutely nothing. Even if you just took literally a second to cut n paste the answer you just gave me above into the project Readme.md it would be helpful to visitors, right?

**Comment by BoPeng** [comment]
Absolutely. We used emails and ticket tracker for discussions and have not committed anything. I will write a readme.
